pages/linux/nsenter.md

```diff
@@ -4,14 +4,18 @@
 > Particularly useful for docker images or chroot jails.
 > More information: <https://github.com/jpetazzo/nsenter/>.
 
-- Run command in existing processes network namespace:
+- Run command using all the same namespaces as an existing process:
```

Comment:
```suggestion
- Run a specific command using all the same namespaces as an existing process:
```

Reply:
I'd also remove the `all` from that line.